### ISSUE TITLE
SFX Update (For AL Specifically)

### ISSUE THREAD

**Issue by Aces6469** [gpstar81/GPStar-proton-pack#365]
### What would you like to add in terms of software changes?

Ok so basically, I was wondering if in 6.0.0 AL mode could be updated to use the 1984 startup sound file that was added recently instead of 001_bootup. Very specifically for AL mode only, as if the brass pack mode doesn't get added separate from FE mode, doing this is gonna break how my SFX is set up. This might be nullified as a possibility when the cleanup happens for audio, but I thought I'd ask anyways.

Short version: Change ONLY AL mode to pull on 379_gb_1984_boot_up instead of 001_bootup when startup is done via the wand.

### Would this request involve any specific hardware?

None

### Homework Completed

- [X] I did my part!

**Comment by nomakewan** [comment]
Starting with 6.0.0, each mode will have its own SFX rather than sharing SFX between modes in order to make end-user customization possible. This was requested in #355 and I fully intend for that to be our resolution.

Closing as duplicate, but know that we already do intend on both having each year have separate WAV files, and that we intend for the brass pack to be its own thing rather than just FE with the lid off.
